vms/platformvm/genesis/camino.go

```diff
@@ -24,3 +29,53 @@ type DepositOffer struct {
 	MinDuration              uint32 `serialize:"true" json:"minDuration"`
 	MaxDuration              uint32 `serialize:"true" json:"maxDuration"`
 }
+
+type MultisigAlias struct {
+	Alias     ids.ShortID   `serialize:"true" json:"alias"`
+	Threshold uint32        `serialize:"true" json:"threshold"`
+	Addresses []ids.ShortID `serialize:"true" json:"addresses"`
+}
+
+func NewMultisigAlias(txID ids.ID, addresses []ids.ShortID, threshold uint32) (MultisigAlias, error) {
+	var err error
+	sortedAddrs := addresses
+	ids.SortShortIDs(sortedAddrs)
+
+	if len(sortedAddrs) < int(threshold) {
+		return MultisigAlias{}, fmt.Errorf("threshold %d is greater than the number of addresses %d", threshold, len(sortedAddrs))
+	}
+
+	ma := MultisigAlias{
+		Addresses: sortedAddrs,
+		Threshold: threshold,
+	}
+
+	ma.Alias, err = ma.ComputeAlias(txID)
+
+	return ma, err
+}
+
+func (ma *MultisigAlias) ComputeAlias(txID ids.ID) (ids.ShortID, error) {
+	// double check passed addresses are sorted, as required
```

Comment:
Because we'll sooner or later send this struct over api network, it should be registered (RegisterType).
If you have done this, you can use simply codec.Marshall and get the bytes.

But this all depends on the MultiSigAlias and MultiSigOwner question -> answer

Reply:
I was unable to register the `MultisigAlias` explicitly in codec because it is placed deep in platformvm code (circular references issue)
However it is serializable by the GenesisCodec as well as entire genesis is.
Using codec's generated bytes for Alias calculation is handy but might be harder to reimplement the calculation on the other platform / language (I'm not aware do JS sdk has codec?)